src/index.ts

```diff
@@ -234,7 +411,7 @@ function isValidItemType(info: PortalItemResponse): boolean {
 }
 
 // Gets a bearer token from the service.
-async function getBearerToken(
+export async function getBearerToken(
```

Comment:
Let's rename this to `getRunToken`. It is a bearer token, but what is important to the user of the API is that this is a token for running reports.